src/accounts/AccountPage.tsx

```diff
@@ -105,7 +105,7 @@ const AccountAboutPage: FC = () => {
   const showDeactivateAccountSection = isFlagEnabled('freeAccountCancellation')
 
   return (
-    <AccountTabContainer activeTab="about">
+    <AccountTabContainer activeTab="settings">
```

Comment:
Since this tab is called `Settings` in the UI, its name in the code should match.